tests/e2e/Event.test.ts

```diff
@@ -13,59 +13,70 @@ describe('e2e', () => {
+        tl1.trustline.prepareUpdate(network1.address, user2.address, 1000, 500),
+        tl2.trustline.prepareUpdate(network1.address, user1.address, 500, 1000)
+      ])
+      await Promise.all([
+        tl1.trustline.confirm(tx1.rawTx),
+        tl2.trustline.confirm(tx2.rawTx)
+      ])
+      // wait for tx to be mined
+      await wait()
     })
 
     describe('#get()', () => {
+      before(async () => {
+        const { rawTx } = await tl1.payment.prepare(network1.address, user2.address, 1.5)
+        await tl1.payment.confirm(rawTx)
+        await wait()
+      })
 
-      it('should return transfer updates', (done) => {
-        tl1.payment.prepare(networkAddress, user2.address, 1.5)
-          .then(({rawTx}) => tl1.payment.confirm(rawTx))
-          .then(() => tl1.event.get(networkAddress))
-          .then((events) => {
-            const last = events[events.length - 1]
-            expect(last.type).to.equal('Transfer')
-            expect(last.direction).to.equal('sent')
-            expect(last.from).to.equal(user1.address)
-            expect(last.to).to.equal(user2.address)
-            expect(last.blockNumber).to.be.a('number')
-            expect(last.timestamp).to.be.a('number')
-            expect(last.address).to.equal(user2.address)
-            expect(last.networkAddress).to.equal(networkAddress)
-            expect(last.status).to.be.a('string')
-            expect(last.amount).to.have.keys('raw', 'value', 'decimals')
-            done()
-          })
+      it('should return latest transfer', async () => {
+        const events = await tl1.event.get(network1.address)
+        const last = events[events.length - 1]
+        expect(last.type).to.equal('Transfer')
+        expect(last.direction).to.equal('sent')
+        expect(last.from).to.equal(user1.address)
+        expect(last.to).to.equal(user2.address)
+        expect(last.blockNumber).to.be.a('number')
+        expect(last.timestamp).to.be.a('number')
+        expect(last.address).to.equal(user2.address)
+        expect(last.networkAddress).to.equal(network1.address)
+        expect(last.status).to.be.a('string')
+        expect(last.amount).to.have.keys('raw', 'value', 'decimals')
+        expect(last.amount.value).to.eq('1.5')
       })
     })
 
     describe('#getAll()', async () => {
       before(async () => {
         const [ txObj1, txObj2 ] = await Promise.all([
-          tl1.trustline.prepareUpdate(networkAddress2, user2.address, 1000, 500),
-          tl2.trustline.prepareUpdate(networkAddress2, user1.address, 500, 1000)
+          tl1.trustline.prepareUpdate(network2.address, user2.address, 1000, 500),
+          tl2.trustline.prepareUpdate(network2.address, user1.address, 500, 1000)
         ])
         await Promise.all([
           tl1.trustline.confirm(txObj1.rawTx),
           tl2.trustline.confirm(txObj2.rawTx)
         ])
-        await new Promise(resolve => setTimeout(() => resolve(), 1000))
+        await wait()
```

Comment:
is 2000 ms enough?

Reply:
yes most of the time this should be enough :)